### ISSUE TITLE
Why is the accuracy of pre-trained models very low?

### ISSUE THREAD

**Issue by zw20yz** [Uason-Chen/CTR-GCN#17]
i download the pretrained model file, and run testing command.but the accurate is low. Is my usage way is wrong?
![image](https://user-images.githubusercontent.com/29089625/146864972-3c6ded54-06ff-45b3-95ed-ec8484ea2c1e.png)

![image](https://user-images.githubusercontent.com/29089625/146864876-72be3c0d-3210-4a02-bcec-fa973bccb3bb.png)

**Comment by Uason-Chen** [comment]
You need to add argument `--test_feeder_args bone=True` in the command to test the model trained with bone data.

**Comment by zw20yz** [comment]
thanks, is work.

**Comment by divyanshusharma1709** [comment]
Hi! I'm having the same issue. Could you help me out? I run this command:

`python main.py --config config/nturgbd120-cross-subject/default.yaml --work-dir . --test-feeder-args bone=True --phase test --save-score True --weights runs-60-59040.pt --device 0`

And this is what I get:
Accuracy:  0.007234410631438233  model:
[ Fri Mar  4 18:19:33 2022 ]    Mean test loss of 795 batches: 11.912733788160407.
[ Fri Mar  4 18:19:34 2022 ]    Top1: 0.72%
[ Fri Mar  4 18:19:34 2022 ]    Top5: 3.92%

**Comment by zw20yz** [comment]
> 

hello, my final commond is python main.py --config config/nturgbd120-cross-subject/default.yaml --work-dir .  --phase test --save-score True --test-feeder-args bone=True --weights runs-60-59040.pt --device 0. test-feeder-args need to be placed in the correct position
